x-pack/plugins/security_solution/public/common/components/events_viewer/event_details_flyout.tsx

```diff
@@ -34,35 +37,42 @@ const EventDetailsFlyoutComponent: React.FC<EventDetailsFlyoutProps> = ({
   browserFields,
   docValueFields,
   timelineId,
+  handleCloseExpandedEvent,
```

Comment:
when adding new props make sure to add them to React.memo function as well https://github.com/elastic/kibana/pull/83963/files#diff-1b758d645e388dc7d27a9fd9707c425b2ee14f9b6ff9f38e07fd302d17529df4R88